v2/pkg/engine/resolve/resolve.go

```diff
@@ -834,6 +834,32 @@ func (r *Resolver) ResolveGraphQLSubscription(ctx *Context, subscription *GraphQ
 		msg := []byte(`{"errors":[{"message":"invalid input"}]}`)
 		return writeFlushComplete(writer, msg)
 	}
+
+	if ctx.ExecutionOptions.SkipLoader {
+		t := newTools(r.options, r.allowedErrorExtensionFields, r.allowedErrorFields)
```

Comment:
Add a comment here explaining what we're doing and why.

Reply:
Added